tests/acceptance/config/behat.yml

```diff
@@ -168,6 +168,22 @@ default:
         - TrashbinContext:
         - GraphContext:
 
+    apiEmailNotification:
+      paths:
+        - '%paths.base%/../features/apiEmailNotification'
+      context: *common_ldap_suite_context
+      contexts:
+        - NotificationContext:
+        - SpacesContext:
+        - FeatureContext: *common_feature_context_params
+        - WebDavPropertiesContext:
+        - OCSContext:
+        - GraphContext:
+        - TrashbinContext:
+        - FavoritesContext:
+        - ChecksumContext:
+        - FilesVersionsContext:
+        - RoleAssignmentContext:
```

Comment:
required all?

Reply:
yes

Reply:
I doubt because the suite doesn't have scenarios related to theses contexts:
```
- WebDavPropertiesContext:
- TrashbinContext:
- FavoritesContext:
- ChecksumContext:
- FilesVersionsContext:
- RoleAssignmentContext:
```
I suspect `SpacesContext` is forcing others to be included. That can be refactored later